migrate/migrations/20190807210857-set-default-system-limits.sql

```diff
@@ -0,0 +1,15 @@
+-- +migrate Up
+INSERT INTO config_limits (id, max)
+    VALUES 
```

Comment:
When I run `make regendb` I get this error:
```
ERRO[0002] perform migration after resettting: pq: syntax error at or near "    VALUES" 
exit status 1
```
Can anyone replicate this? @dayander , @m17ch , @mastercactapus

Reply:
Yes I get the same error